client/packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -250,8 +273,8 @@ export const useDialog = (dialogProps?: DialogProps): DialogState => {
           <DialogActions
             sx={{
               justifyContent: 'center',
-              marginBottom: '30px',
-              marginTop: '30px',
+              marginBottom: keyboardIsOpen ? 0 : '30px',
+              marginTop: keyboardIsOpen ? 0 : '30px',
```

Comment:
Makes the margins around the buttons smaller while using the keyboard (because they take up a lot of space)